django/pyrog/management/commands/import_mappings.py

```diff
@@ -0,0 +1,29 @@
+import json
+import logging
+from glob import glob
+
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+from pyrog.api.serializers.mapping import MappingSerializer
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        # TODO doc
```

Comment:
this is a must have. I spent too much time figuring out exactly how it worked when you left (especially the checkout mechanism)